presto-hudi/src/main/java/com/facebook/presto/hudi/HudiTableType.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.hudi;
+
+public enum HudiTableType
+{
+    COW,
+    MOR,
+    UNKNOWN,
+    /**/;
+
+    public static HudiTableType fromInputFormat(String inputFormat)
```

Comment:
Won't the table meta client give us the table type?

Reply:
For a MOR tables hudi-sync-tools would create two table entries for  it, one is `_ro` and the other is `_rt`, they have different input format names stored in metastore, and the hive compatible engines are expected to process them in different read paths. So we need to determine the table type by its input format.